nestkernel/event_delivery_manager.cpp

```diff
@@ -706,7 +701,6 @@ EventDeliveryManager::gather_target_data( const thread tid )
     if ( gather_completed_checker_.all_true() )
     {
       set_complete_marker_target_data_( assigned_ranks, send_buffer_position );
-#pragma omp barrier
     }
```

Comment:
Agree here: Superfluous barrier because of preceding invocation of gather_completed_checker_.all_true().